components/announcement_bar/configuration_bar/configuration_bar.tsx

```diff
@@ -157,148 +157,165 @@ class ConfigurationAnnouncementBar extends React.PureComponent<Props> {
+                    type={AnnouncementBarTypes.CRITICAL}
+                    message={
+                        <>
+                            <img
+                                className='advisor-icon'
+                                src={warningIcon}
                             />
-                        }
-                    />
-                );
-            }
+                            <FormattedMessage
+                                id='announcement_bar.error.license_expired'
+                                defaultMessage='Enterprise license is expired and some features may be disabled.'
+                            />
+                            <RenewalLink/>
+                        </>
+                    }
+                />
+            );
+        }
 
-            if (isLicenseExpired(this.props.license)) {
-                return (
-                    <AnnouncementBar
-                        type={AnnouncementBarTypes.CRITICAL}
-                        message={
-                            <FormattedMarkdownMessage
-                                id={AnnouncementBarMessages.LICENSE_EXPIRED}
-                                defaultMessage='Enterprise license is expired and some features may be disabled. [Please renew](!{link}).'
-                                values={{
-                                    link: renewalLink,
-                                }}
+        if (isLicenseExpired(props.license)) {
+            return (
+                <AnnouncementBar
+                    type={AnnouncementBarTypes.CRITICAL}
+                    message={
+                        <>
+                            <img
+                                className='advisor-icon'
+                                src={warningIcon}
                             />
-                        }
-                    />
-                );
-            }
+                            <FormattedMessage
+                                id='announcement_bar.error.license_expired'
+                                defaultMessage='Enterprise license is expired and some features may be disabled.'
+                            />
+                            <RenewalLink/>
+                        </>
+                    }
+                />
+            );
+        }
 
-            if (isLicenseExpiring(this.props.license) && !this.props.dismissedExpiringLicense) {
-                return (
-                    <AnnouncementBar
-                        showCloseButton={true}
-                        handleClose={this.dismissExpiringLicense}
-                        type={AnnouncementBarTypes.ANNOUNCEMENT}
-                        message={
-                            <FormattedMarkdownMessage
-                                id={AnnouncementBarMessages.LICENSE_EXPIRING}
-                                defaultMessage='Enterprise license expires on {date, date, long}. [Please renew](!{link}).'
+        if (true) {
```

Comment:
I believe this was an unintended change?

Reply:
yes! we found it with @nickmisasi couple of minutes ago too!